frontend/client/src/app/components/ConfirmDialog.tsx

```diff
@@ -15,6 +15,7 @@ export interface ConfirmDialogProps {
     | "danger"
     | "warning"
     | "info"
+    /* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
I saw others eslint inline here and there and again I think those things will be solved in the future so no problem for me.